### PR TITLE
update Core Developers and EmCDs

### PR DESCRIPTION
**Pull request by orbeckst**
- result of 2021 role call
  - updated list of Core Devs
  - add new list of EmCDs
- added subsections for CDs and EmCDs with short summary of function
  (based on the passed proposal for EmCDs)
- renamed "Decision Making Process" to "Decision Making Process and Membership"
- added 3rd item to "Decision Making Process and Membership" on CD opt-in and EmCD

### CODE REVIEW

**Review by IAlibay — APPROVED** [review]
lgtm thanks!

**Comment by IAlibay** [comment]
@orbeckst hold for results of the current vote? We've already passed majority votes - just giving it a little bit more time so the last person gets to vote.

**Comment by orbeckst** [comment]
Sure—could you update the PR, please?

I don’t know if anybody else wants to formally ok it—for anything that changes the governance section i looks better when at least a few coredevs agreed.

File: about.md [hunk 1]
```diff
@@ -46,44 +46,66 @@ Installable packages are available through the popular ``pip`` and
...
+merges.
+
+The current *Emeriti Core Developers* are:
+
+- @dotsdl
+- @jandom
+- @kain88-de
 - @micaela-matta
 - @mnmelo
 - @mtiberti
```
Comment (lilyminium) [inline about.md:96]:
```suggestion
- @mtiberti
- @nmichaud
```

**Review by lilyminium — APPROVED** [review]
LGTM, including with Naveen and Elizabeth additions to Emeriti.

File: about.md (hunk 1 above)
Comment (IAlibay) [reply about.md:97]: Done :) [sorry I was already on an edit page so didn't see your suggestion]

**Comment by orbeckst** [comment]
fyi, Liz says that @denniej0-2 @denniej0 and @edenning are all her handles but she has to figure out which one she still knows the password for and which one to link to her email address. For right now she's happy to be listed her with her real name, which can be changed to the handle at a later time.

**Comment by orbeckst** [comment]
@IAlibay I'll let you merge whenever you're happy. Thank you!

**Comment by orbeckst** [comment]
Thanks, I am updating NF by email in a second.

**Pull request merged**